krabs/krabs/trace.hpp

```diff
@@ -319,6 +343,16 @@ namespace krabs {
         properties_.LogFileMode = properties->LogFileMode;
     }
 
+    template <typename T>
+    void trace<T>::set_trace_information(
+        TRACE_INFO_CLASS informationClass,
```

Comment:
let's use snake_case for these parameter names

Reply:
Done.